ksqldb-api-client/pom.xml

```diff
@@ -158,6 +158,12 @@
             <version>${jetty.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.8.6</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
trying to figure out a replacement for this. Doing  `final JsonArray values = new JsonArray(dataRow.getColumns())` instead of using gson leads to `Illegal type in JsonObject: class java.math.BigDecimal`

Reply:
Before your change, when we were using the Vertx json directly, what type does it use for a BigDecimal equivalent?